application/templates/static_site/measure.html

```diff
@@ -539,7 +547,11 @@ <h2 id="download-the-data" class="govuk-accordion__section-heading">
             {% for file in measure_version.uploads  %}
               {% if file.file_name %}
                 {% set measure_version_data_url =
-                  ("/" + topic_slug + "/"+  subtopic_slug + "/"+  measure_version.measure.slug + "/"+  version + "/downloads/" + file.file_name)
```

Comment:
:see_no_evil:  why were these links ever this way?

Are we sure they will work for the static build? (I can't think why they wouldn't... just trying to figure out why they might be like this in the first place).

Reply:
I'm pretty sure I ran the static build and checked these links and they seemed fine. I'll double-check again, just in case...